undertow/src/main/resources/schema/wildfly-undertow_3_0.xsd

```diff
@@ -535,5 +535,6 @@
         <xs:attribute name="request-queue-size" type="xs:int" use="optional" />
         <xs:attribute name="security-realm" type="xs:string" use="optional" />
         <xs:attribute name="use-alias" type="xs:string" use="optional" default="false" />
+        <xs:attribute name="enable-http2" type="xs:string" use="optional" default="false" />
```

Comment:
Ah ok, numerous other subsystems actually use the "right" type in XSD, but then use of expressions fail validation of course, rendering those XSD useless. So your way is better.